repondeur/zam_repondeur/models/texte.py

```diff
@@ -85,6 +85,12 @@ class Texte(Base):
         "date_depot",
     )
 
+    @property
+    def session_str(self) -> Optional[str]:
+        if self.session is None:
+            return None
```

Comment:
If we don't return an empty string we will have `None` in URLs too?

Reply:
For AN, `Texte.session` is `None` but we have `Texte.legislature`. For Sénat, it's the opposite. So we should never have None in the Sénat fetch URLs.

Reply:
👌